### PR TITLE
Feature/86cuvrq0c user sign up

### PR DESCRIPTION
**Pull request by RyanAHicks**
Completed ticket number: https://app.clickup.com/t/86cuvrry3

To check, please go to: http://localhost:3000/register

The user should firstly select Account Type (Bride or Artist).

Secondly, the user should enter details to create an account - username, name, email, password, retype password.
- Username and email must be unique (can't enter the same username and/or email to make multiple accounts).
- Ensure password adheres to validation requirements.
- Ensure retype-password matches password.

Thirdly, enter the 6-digit code (NOTE: email API not yet implemented - enter any 6 digits for now)

Fourthly, account has been signed up, can return the login page.

### CODE REVIEW

**Review by hiri45 — APPROVED** [review]
implementation meets standards, with result representing what is displayed on the figma. required tests have been passed.

File: behind-the-veil-siteroot/imports/ui/App.jsx
Comment (hiri45) [inline behind-the-veil-siteroot/imports/ui/App.jsx:1]: There is a small conflict im pretty sure which is preventing me from merging, can you please resolve this for me to push. Thanks!!

Comment (RyanAHicks) [reply behind-the-veil-siteroot/imports/ui/App.jsx:1]: Just conducted the rebase and should be all good (I think). Let me know if you encounter any other issues!

**Review by hiri45 — APPROVED** [review]

**Comment by RyanAHicks** [comment]
I believe I have now resolved the conflict in App.jsx

**Review by hiri45 — APPROVED** [review]

**Pull request merged**